base/libgit2/types.jl

```diff
@@ -449,6 +449,16 @@ struct FetchHead
     ismerge::Bool
 end
 
+function Base.show(io::IO, fh::FetchHead)
+    println(io, "FetchHead:")
+    println(io, "Name: $(fh.name)")
+    println(io, "URL: $(fh.url)")
+    print(io, "OID: ")
+    show(io, fh.oid)
```

Comment:
I wanted to use `show` just in case `GitHash` has a nice `show` method defined, which I think it does. I did this by analogy with the linalg `show` methods.